loadData/book_data.rb

```diff
@@ -0,0 +1,16 @@
+require 'json'
+require_relative '../school/book'
+
+@file_book = File.read('./data/book.json')
+
+def book_data(book)
+  return if @file_book.empty?
+
+  data_book = JSON.parse(@file_book)
+  data_book.map do |value|
+    title = value['title']
+    author = value['author']
+    book_data = Book.new(title, author)
+    book << book_data
+  end
+end
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/176337255-d85590df-0a8e-47c6-b38b-42d1d4bc4e29.png)

- [ ] Great job on saving data in JSON files, However, in the requirements it said that you should handle the case where one of the JSON files is missing the application should not throw an error. I deleted the `/data/book.json` file and the app crashed. Check this hint solution :point_down: 

```rb
# Kindly note that this is just an example solution, feel free to solve it the way you want, just make sure the app does nit crash when this file is missing
require 'json'
require_relative '../school/book'


def book_data(book)
  File.write('./data/book.json', JSON.pretty_generate([])) unless File.exists?('./data/book.json')
  @file_book = File.read('./data/book.json')
  return if @file_book.empty?

  data_book = JSON.parse(@file_book)
  data_book.map do |value|
    title = value['title']
    author = value['author']
    book_data = Book.new(title, author)
    book << book_data
  end
end
```

Reply:
@andersonlebon Thank you for the feedback, I will update the project.